src/collection.js

```diff
@@ -20,15 +25,47 @@ CKEDITOR.define( [ 'emittermixin', 'ckeditorerror', 'utils' ], function( Emitter
 		 * Creates a new Collection instance.
 		 *
 		 * @constructor
+		 * @param {Iterale} [items] Items to be added to the collection.
+		 * @param {String} [idName='id'] The name of the property which is considered to identify an item.
 		 */
-		constructor() {
+		constructor( items, idName ) {
```

Comment:
Yes, I would propose that. Actually, there are two alternatives:

1. `Collection( items, options )`
2. `Collection()` + configuration property + `add( item | items )`

Both are ok for me.